### DISCUSSION TITLE
Can anyone upload next-site repository here?

### DISCUSSION THREAD

**Discussion by mentalweb** [vercel/next.js#20328] [Help]
Hi there.
I want to fork https://github.com/vercel/next-site repo (nextjs.org). but last few days I can't see this repo here. Can anyone upload the code here?

**Comment by waptik** [comment]
It's closed source. You find the old version(pre nextjs conf) on github if you search enough.

**Reply by mentalweb** [reply]
I searched a lot for this repo but I didn't find anything. :(
how can they private that repo like that? :(

**Comment by mentalweb** [comment]
I searched a lot for this repo but I didn't find anything. :(
how can they private that repo like that? :(

**Reply by waptik** [reply]
They've added lots of stuffs because of the conference . If you are just looking for the docs, you can check https://github.com/vercel/next.js/tree/canary/docs.
Other than that, you'll have to wait until they make it open source again
